### PR TITLE
Fix sklearn metric

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do?
Fixes https://github.com/PyTorchLightning/pytorch-lightning/pull/1327#issuecomment-633732609

## PR review    
Anyone in the community is free to review the PR once the tests have passed.     
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

**Comment by Borda** [comment]
merged directly as eda08ca04ffc017e10dde364acb2e13f17ff65b8
